test/index.spec.js

```diff
@@ -110,11 +104,11 @@ describe('InlineSourceWebpackPlugin', () => {
         // inline.js
         `<script>function Person(){}Person.prototype.sayHello=function(){console.log("[inline]:","hello world!")},(new Person).sayHello();</script>`,
         // webpack runtime file
-        `window.webpackJsonp=window.webpackJsonp||[]`,
+        `webpack`,
         // bundle.js
         `console.log("This file is build by webpack.But InlineSourceWebpackPlugin will embed it into html file.")`,
         // appended by html-webpack-plugin
-        /<script src="\/inline-source-webpack-plugin\/dist\/index\.\w+\.js"><\/script>/
+        /<script (?:defer="defer" )?src="\/inline-source-webpack-plugin\/dist\/index\.\w+\.js"><\/script>/
```

Comment:
New versions of `html-webpack-plugin` include the `defer="defer"` attribute on the entry script tag.
This test adds an optional clause to the regexp to allow the entry script tag to include, or not include this attribute.